lib/challenge.js

```diff
@@ -229,10 +257,10 @@ let nums = [
 ];
 
 function prompt11() {
-  let median;
-  // YOUR CODE HERE
+  let median = Math.floor(nums.length/2);
+  console.log(nums[median]);
 
-  console.log(median);
+  // console.log(median);
 }
 // COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
 // prompt11();
```

Comment:
Watch the spacing on line 260.  try to have spaces between operators and their operands.
I would rewrite it as:
`let median = Math.floor(nums.length / 2)`

This is a good start. Challenge yourself to complete it. (resubmit not required)

> Find the median number in the following nums array (the mid point), then
> console.log that number. The median number is the middle number in a SORTED list of numbers. Think through the steps you'll need to take to determine this number with code!